File: 1-two-sum/two-sum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target, int* returnSize) {

     int* result = (int*)malloc(2 * sizeof(int));
    int i;
    int second_number,j;
    for(i=0;i<numsSize;i++){
        second_number=target-nums[i];
        for(j=i+1;j<numsSize;j++){
            if(second_number==nums[j]){
                result[0]=i;
                result[1]=j;
                *returnSize=2;
                return result;
            }
        }
    }
    *returnSize=0;
    return NULL;
    
}

File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        result=[]
        sec_num=0
#         for i in range (len(nums)):
#             sec_num=target-nums[i]
#             for j in range(i+1,len(nums)):
#                 if(nums[j]==sec_num):
#                     result.extend([i,j])
#                     return result
        seen={}
        for i,num in enumerate(nums):
            sec_num=target-num
            if sec_num in seen:
                return [seen[sec_num],i]
            seen[num]=i
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        